codesign/bin/codesign.dart

```diff
@@ -44,58 +50,65 @@ Future<void> main(List<String> args) async {
     )
     ..addOption(
-      kCodesignTeamId,
+      kCodesignTeamIdOption,
       help: 'Team-id is used by notary service for xcode version 13+.',
     )
     ..addOption(
-      kCommit,
+      kCommitOption,
       help: 'the Flutter engine commit revision for which Google Cloud Storage binary artifacts should be codesigned.',
     )
     ..addMultiOption(
-      kCodesignFilepath,
+      kCodesignFilepathOption,
       help: 'The zip file paths to be codesigned. Pass this option multiple'
           'times to codesign multiple zip files',
       valueHelp: 'darwin-x64/font-subset.zip',
     )
     ..addFlag(
-      kProduction,
+      kProductionFlag,
       help: 'whether we are going to upload the artifacts back to GCS for production',
     );
 
   final ArgResults argResults = parser.parse(args);
 
-  final Platform platform = LocalPlatform();
+  const Platform platform = LocalPlatform();
 
-  final String commit = getValueFromEnvOrArgs(kCommit, argResults, platform.environment)!;
-  final String codesignCertName = getValueFromEnvOrArgs(kCodesignCertName, argResults, platform.environment)!;
-  final String codesignUserName = getValueFromEnvOrArgs(kCodesignUserName, argResults, platform.environment)!;
-  final String appSpecificPassword = getValueFromEnvOrArgs(kAppSpecificPassword, argResults, platform.environment)!;
-  final String codesignAppstoreId = getValueFromEnvOrArgs(kCodesignAppStoreId, argResults, platform.environment)!;
-  final String codesignTeamId = getValueFromEnvOrArgs(kCodesignTeamId, argResults, platform.environment)!;
+  final String commit = getValueFromEnvOrArgs(kCommitOption, argResults, platform.environment)!;
+  final String codesignCertName = getValueFromEnvOrArgs(kCodesignCertNameOption, argResults, platform.environment)!;
+  final String codesignUserName = getValueFromEnvOrArgs(kCodesignUserNameOption, argResults, platform.environment)!;
+  final String appSpecificPassword =
+      getValueFromEnvOrArgs(kAppSpecificPasswordOption, argResults, platform.environment)!;
+  final String codesignAppstoreId = getValueFromEnvOrArgs(kCodesignAppStoreIdOption, argResults, platform.environment)!;
+  final String codesignTeamId = getValueFromEnvOrArgs(kCodesignTeamIdOption, argResults, platform.environment)!;
 
-  final List<String> codesignFilepaths = argResults[kCodesignFilepath]!;
-  final bool production = argResults[kProduction] as bool;
+  final List<String> codesignFilepaths = argResults[kCodesignFilepathOption]!;
+  final bool production = argResults[kProductionFlag] as bool;
 
   if (!platform.isMacOS) {
-    throw ConductorException(
+    throw CodesignException(
       'Error! Expected operating system "macos", actual operating system is: '
       '"${platform.operatingSystem}"',
     );
   }
 
+  FileSystem fileSystem = const LocalFileSystem();
+  Directory tempDir = fileSystem.systemTempDirectory.createTempSync('conductor_codesign');
+  Logger logger = Logger('codesign-logger');
```

Comment:
nit: `Logger('codesign')` (this gets prepended to the logs)

Reply:
Sure, renamed.